doc/source/conf.py

```diff
@@ -569,7 +569,11 @@ def linkcode_resolve(domain, info):
             return None
 
     try:
-        fn = inspect.getsourcefile(obj)
+        # inspect.unwrap() was added in Python version 3.4
+        if sys.version_info >= (3, 4):
```

Comment:
For a review like this I think it's useful the new github feature to suggest changes (the icon with the +- icon when adding a comment. This way, afaik, the user can make the change with a single click.

```suggestion
        if sys.version_info >= (3, 5):
```